sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala

```diff
@@ -592,7 +607,10 @@ case class Multiply(
   override def inputType: AbstractDataType = NumericType
 
   override def symbol: String = "*"
-  override def decimalMethod: String = "$times"
+
+  override def decimalMethod(mathContextValue: GlobalValue, value1: String, value2: String):
+  String = s"Decimal.apply($value1.toJavaBigDecimal()" +
+    s".multiply($value2.toJavaBigDecimal(), $mathContextValue))"
```

Comment:
Adjust indentation according to https://github.com/databricks/scala-style-guide#indent